src/mscorlib/src/System/Resources/ResourceManager.cs

```diff
@@ -465,7 +463,15 @@ public virtual String BaseName
         public virtual bool IgnoreCase
         {
             get { return _ignoreCase; }
-            set { _ignoreCase = value; }
+            set 
+            { 
+                if (_PRIonAppXInitialized && value)
```

Comment:
Is this more broadly breaking than we had discussed? This would mean that a NS2.0 library isn't allowed to set IgnoreCase to true. I thought the idea was that this should only happen if there were PRI resources matching BaseName.